subprojects/core/src/main/java/org/gradle/api/internal/artifacts/configurations/ConfigurationRolesForMigration.java

```diff
@@ -39,7 +39,13 @@ public enum ConfigurationRolesForMigration implements ConfigurationRole {
     @Deprecated
     INTENDED_RESOLVABLE_BUCKET_TO_INTENDED_RESOLVABLE(ConfigurationRoles.INTENDED_RESOLVABLE_BUCKET, ConfigurationRoles.INTENDED_RESOLVABLE),
     @Deprecated
-    INTENDED_CONSUMABLE_BUCKET_TO_INTENDED_CONSUMABLE(ConfigurationRoles.INTENDED_CONSUMABLE_BUCKET, ConfigurationRoles.INTENDED_CONSUMABLE);
+    INTENDED_CONSUMABLE_BUCKET_TO_INTENDED_CONSUMABLE(ConfigurationRoles.INTENDED_CONSUMABLE_BUCKET, ConfigurationRoles.INTENDED_CONSUMABLE),
+
+    /**
+     * A resolvable bucket that will become a bucket in the next major version.
+     */
+    @SuppressWarnings("deprecation")
+    RESOLVABLE_BUCKET_TO_BUCKET(ConfigurationRoles.INTENDED_RESOLVABLE_BUCKET, ConfigurationRoles.INTENDED_BUCKET);
```

Comment:
❕ Isn't this a duplicate of line 40?

Reply:
40 goes to resolvable. This goes to bucket

Reply:
D'oh!  Yes it does.